content/desktop/hardened-desktop/settings-management/air-gapped-containers.md

```diff
@@ -0,0 +1,86 @@
+---
+description: Learn how to create air-gapped containers with Settings Management
+title: Configure air-gapped containers with Settings Management
+keywords: settings management, air gapped, security, Docker Desktop, configuration, proxy, network
+---
+
+> **Beta feature**
+>
+> This feature is in [Beta](../../../release-lifecycle.md/#beta).
+> It's available with Docker Desktop version 4.29 and later.
+{ .experimental }
+
+Air-gapped containers allows administrators to restrict containers from accessing network resources, limiting where data can be uploaded to or downloaded from.
+
+Docker Desktop can apply a custom set of proxy rules to network traffic from containers. The proxy can be configured to:
+
+- Allow network connections
+- Reject network connections
+- Tunnel through an HTTP or SOCKS proxy
+
+You can choose:
+
+- Which outgoing TCP ports the policy applies to. For example, only certain ports, `80`, `443` or all with `*`.
+- Whether to forward to a single HTTP or SOCKS proxy, or to have a policy per destination via a Proxy Auto-Configuration (PAC) file.
+
+## Configuration
+
+Assuming enforced sign-in and Settings Management are enabled, add the new proxy configuration to the `admin-settings.json` file. For example:
```

Comment:
Maybe we could link to the docs for enforced sign-in and settings management here?